src/components/ArticleList.jsx

```diff
@@ -13,15 +14,15 @@ function ArticleList() {
     getArticles();
   }, []);
   return (
-    <div className="article-list">
+    <Stack spacing={1}>
       {articles.length > 0 ? (
```

Comment:
I think I mentioned this before, but an isLoading pattern is often the better way of dealing with a conditional render like this one. It's more readable for any other developer coming to your code.